### ISSUE TITLE
data

### ISSUE THREAD

**Issue by rongtongxueya** [nkszjx/UTB_master#1]
great job！if it's convenient for you, could you please share your test and training sample data with me? I'd like to learn by running the code. Thank you very much for your help.

**Comment by nkszjx** [comment]
Some training data samples are available at "Train_dataset.zip"

**Comment by rongtongxueya** [comment]
Thank you for your response, but I have a new question now. It seems that in train.py, the 'scale' and 'mirror' parameters are not defined in the VOCDataSet function: train_dataset = VOCDataSet(args.data_dir, args.data_list, crop_size=input_size, scale=args.random_scale, mirror=args.random_mirror, mean=IMG_MEAN). Moreover, in evaluation.py during testing, it seems the output is three values:

for index, batch in enumerate(testloader):
if index % 100 == 0:
print('%d processed'%(index))
image, size, name = batch
size = size[0]
with torch.no_grad():
output,_ = model(Variable(image).cuda()) # model test

Yet, another question I have is regarding how the confusion matrix, MIoU, Kappa, OA, PA, and UA are outputted in your code. I tried to write a piece of code to output the confusion matrix during testing, but I can't seem to determine the labels in the test set.

I deeply apologize for any inconvenience caused, as I am new to this area and hope you can help me with these issues. Thank you in advance.
